src/Console/EventStoreWorker.php

```diff
@@ -1,10 +1,12 @@
 <?php
 
-namespace DigitalRisks\LaravelEventStore;
+namespace DigitalRisks\LaravelEventStore\Console;
```

Comment:
o_O - how was this working to begin with? Also, shouldn't it be in `Console\Commands`